systemtest/src/test/java/io/strimzi/systemtest/kafka/ListenersST.java

```diff
@@ -657,12 +657,15 @@ void testCustomCertLoadBalancerAndTlsRollingUpdate() throws Exception {
         );
 
         internalKafkaClient.setConsumerGroup("consumer-group-certs-71");
-        internalKafkaClient.setMessageCount(MESSAGE_COUNT * 3);
+        internalKafkaClient.setMessageCount(MESSAGE_COUNT);
 
         sent = internalKafkaClient.sendMessagesTls();
-        assertThat(sent, is(3 * MESSAGE_COUNT));
+        assertThat(sent, is(MESSAGE_COUNT));
+
+        internalKafkaClient.setMessageCount(MESSAGE_COUNT * 5);
```

Comment:
Why 5 times?

Reply:
I need to get some info from Maros about clients -> but if you set different CONSUMER_GROUP you'll receive all messages until that moment -> not just messages you set in setMessagesCount. So maybe it's problem in VerifiableClient. But in the original test (before refactor of the clients) was it this way.

Reply:
Can you open the issue for it? We should understand what's happening here.

Reply:
Yeah sure -> how I said - maybe it's fault in understanding from my side, we will see.